frontend/components/ThemeProvider/Theme.ts

```diff
@@ -68,19 +72,26 @@ const warning = {
 const light: ITheme = {
   body: neutral.textInverted,
   text: neutral.text,
-  border: neutral["100"],
+  link: {
+    text: neutral.text,
+    hovered: primary.blue,
+  },
+  fadedText: neutral[700],
+  // border: neutral["100"],
```

Comment:
Is this comment necessary? What's its purpose?

Reply:
That is for my another commit later.

Reply:
Gotcha!